lib/gringotts/gateways/paymill.ex

```diff
@@ -341,7 +344,7 @@ defmodule Gringotts.Gateways.Paymill do
     defp set_success(opts, %{"error" => error}) do
       opts ++ [message: error, success: false]
     end
-    defp set_success(opts, %{"transaction" => %{ "response_code" => 20000}}) do
+    defp set_success(opts, %{"transaction" => %{"response_code" => 20_000}}) do
```

Comment:
I know this follows the style guide. however I want to be skeptical here and I think this unneccesary formating, elixir is ultimatley going to convert this into 20000 so it increase one more processing. Still I would like to know what @ashish173 and @SagarKarwande  thinks over this.

Reply:
IMO it's not unnecessary formatting as it improves readability. Moreover there is no "extra cost" in this as the parser will do the conversion at compile time.

Reply:
@oyeb Thats cool then. lets merge it. 
@ashish173 LGTM :+1: